docs/source/cpp/csv.rst

```diff
@@ -25,15 +25,26 @@ Reading and Writing CSV files
 =============================
 
 Arrow provides a fast CSV reader allowing ingestion of external data
-as Arrow tables.
+as Arrow Tables or streamed as Arrow RecordBatches.
```

Comment:
```suggestion
to create Arrow Tables or a stream of Arrow RecordBatches.
```

This sentence feels slightly grammatically off to me.  It sounds like the input data is tables or batches but really it's the output.

Reply:
Fair enough. Fixed in 381f8b29ba205f6e0040c632eb23cec8eb46764c.